coverage_config_x86_64.json

```diff
@@ -1,5 +1,5 @@
 {
-  "coverage_score": 85.5,
+  "coverage_score": 75.6,
```

Comment:
There is no point in giving you grief over that one, especially since I know how hard it is to deal with the test coverage infrastructure.

I am done reviewing this set.

Reply:
:)

Just as an FYI, the number came down here as I wasn't able to add tests for the libgpiod based implementatoin as it requires a physical device.